src/_locales/en/messages.json

```diff
@@ -0,0 +1,5 @@
+{
+	"extensionName": {
+		"message": "500px On New Tab"
+	}
+}
```

Comment:
If "default_locale" is set to "en", the en translation file must exist.

Reply:
Ok, i create it, ty